packages/suite-analytics/src/types/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './events';
```

Comment:
please move this export directly to `src/index.ts` this is only adding noise to file system